### ISSUE TITLE
TypeError when running mtgstocks.py

### ISSUE THREAD

**Issue by skoogiz** [mtgjson/mtgjson#333]
#### MTGJSON Version: 4.X.X
4.4.0

#### Description of Bug:

Executing `python3 -m mtgjson4 -ac --skip-keys` and results is this:

```
[INFO] 2019-04-29 11:53:32,254: Downloaded: https://api.mtgstocks.com/api/v1/mtgjson?token= (Cache = False)
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/anders.skoglund/development/ask/mtgjson/mtgjson4/__main__.py", line 191, in <module>
    main()
  File "/Users/anders.skoglund/development/ask/mtgjson/mtgjson4/__main__.py", line 165, in main
    compiled = compile_mtg.build_output_file(sf_set, set_code, args.skip_keys)
  File "/Users/anders.skoglund/development/ask/mtgjson/mtgjson4/compile_mtg.py", line 135, in build_output_file
    card_holder = add_stocks_data(card_holder)
  File "/Users/anders.skoglund/development/ask/mtgjson/mtgjson4/compile_mtg.py", line 208, in add_stocks_data
    stocks_data = mtgstocks.get_card_data(card.get("tcgplayerProductId"))
  File "/Users/anders.skoglund/development/ask/mtgjson/mtgjson4/provider/mtgstocks.py", line 97, in get_card_data
    return __get_stocks_data().get(str(tcgplayer_id), None)
  File "/Users/anders.skoglund/development/ask/mtgjson/mtgjson4/provider/mtgstocks.py", line 80, in __get_stocks_data
    save_dictionary[row["tcg_id"]] = row
TypeError: string indices must be integers
```

**Comment by ZeldaZach** [comment]
What version of Python3 are you running? I build and test with 3.7.3

**Comment by skoogiz** [comment]
I also run Python 3.7.3

**Comment by skoogiz** [comment]
I tweeked it to use the same skip as the other price sites to be able to build
```
# Add MTGStocks data in
    if not skip_keys:
        card_holder = add_stocks_data(card_holder)
```

**Comment by ZeldaZach** [comment]
New PR is up if you want to try it out

**Comment by skoogiz** [comment]
I just succesfully finished my build with the same snipet

**Comment by ZeldaZach** [comment]
Fixed in #336
